urllib3/connection.py

```diff
@@ -162,6 +164,38 @@ def connect(self):
         conn = self._new_conn()
         self._prepare_conn(conn)
 
+    def request_chunked(self, method, url, body=None, headers={}):
```

Comment:
We're not using that value at all. We make a copy with `HTTPHeaderDict`. I'm not sure using `None` as a default really buys us much more safety.